app/src/lib/text-token-parser.ts

```diff
@@ -131,8 +131,15 @@ export class Tokenizer {
     index: number,
     repository: GitHubRepository
   ): LookupResult | null {
-    const nextIndex = this.scanForEndOfWord(text, index)
-    const maybeIssue = text.slice(index, nextIndex)
+    let nextIndex = this.scanForEndOfWord(text, index)
+    let maybeIssue = text.slice(index, nextIndex)
+
+    // scanForEndOfWord doesn't detect ')'
```

Comment:
Can we make this comment mention that this is due to the format of "squash and merge" commits, e.g:

```
    // handle situation where issue reference is wrapped in parentheses
    // like the generated "squash and merge" commits on GitHub
    if (maybeIssue.endsWith(')')) {
      nextIndex -= 1
      maybeIssue = text.slice(index, nextIndex)
    }
```